code/game/turfs/simulated/floor/chasm.dm

```diff
@@ -120,6 +120,7 @@
 		AM.forceMove(T)
 		if(isliving(AM))
 			var/mob/living/L = AM
+			L.update_z(drop_z)
 			L.Weaken(10 SECONDS)
 			L.adjustBruteLoss(30)
 	falling_atoms -= AM
```

Comment:
I feel like this should just be handled by forceMove in general. is there ever a time where we wouldn't want this on forcemove?